pkg/cmd/cli/describe/describer.go

```diff
@@ -214,7 +214,12 @@ func describeBuildSpec(p buildapi.BuildSpec, out *tabwriter.Writer) {
 			formatString(out, "Binary", "provided on build")
 		}
 	}
-
+	for _, image := range p.Source.Images {
+		formatString(out, "Files from", fmt.Sprintf("%s/%s", image.From.Namespace, image.From.Name))
```

Comment:
may want to use something like https://github.com/openshift/origin/blob/master/pkg/util/httpproxy/upgradeawareproxy.go#L79-L90 to join namespace and name, otherwise for most cases, you'll have "/imageName" which doesn't look good. I see we could use a function like that elsewhere in this file:
https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/describer.go#L228-L232
https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/describer.go#L257-L261
https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/describer.go#L278-L284
https://github.com/openshift/origin/blob/master/pkg/cmd/cli/describe/describer.go#L300-L306